src/main/java/org/terasology/navgraph/BaseRegion.java

```diff
@@ -20,30 +20,56 @@
 
 /**
  * @author synopia
+ * Abstract class which floors and regions derive from
  */
 public abstract class BaseRegion<N extends BaseRegion> {
     public int id;
     public Set<N> neighborRegions = new HashSet<N>();
     protected BitMap map;
     protected Floor floor;
 
+    /**
+     * Creates a BaseRegion
+     * @param id id of the base region
+     */
     protected BaseRegion(int id) {
         this.id = id;
         map = new BitMap();
     }
 
+    /**
+     * Marks a point in the region as walkable by characters
+     * @param x x co-ordinate of the point with respect to region's top-left corner
```

Comment:
Same here (and elsewhere, I won't mark them all) as for the "id id" comment above